static/app/views/starfish/components/facetInsights.tsx

```diff
@@ -0,0 +1,184 @@
+    {
+      field: 'sumdelta',
+      kind: 'desc',
+    },
+  ]);
+
+  function renderBodyCell(column: GridColumnHeader, row: DataRow): React.ReactNode {
+    if (column.key === 'throughput' || column.key === 'p50') {
+      return (
+        <Sparkline
+          color={column.key === 'throughput' ? CHART_PALETTE[3][0] : CHART_PALETTE[3][1]}
+          series={row[column.key]}
+          width={column.width ? column.width - column.width / 5 : undefined}
+        />
+      );
+    }
+
+    if (column.key === 'tagValue') {
+      const query = new MutableSearch(eventView.query);
+      let queryFilter = '';
+      query.tokens.forEach(value => {
+        if (value.key) {
+          queryFilter = queryFilter.concat(' ', `${value.key}:${value.value}`);
+        }
+      });
+      queryFilter = queryFilter.concat(' ', `${row.tagKey}:${row.tagValue}`);
+      return (
+        <OverflowEllipsisTextContainer>
+          <Link
+            to={`/discover/homepage/?${qs.stringify({
+              ...eventView.generateQueryStringObject(),
+              query: queryFilter,
+              field: [
+                'title',
+                'p50(transaction.duration)',
+                'p75(transaction.duration)',
+                'p95(transaction.duration)',
+              ],
+              yAxis: 'count()',
+            })}`}
+          >
+            {row[column.key]}
+          </Link>
+        </OverflowEllipsisTextContainer>
+      );
+    }
+
+    return <TextAlignLeft>{row[column.key]}</TextAlignLeft>;
+  }
+
+  const {isLoading, data} = useGenericDiscoverQuery<any, DiscoverQueryProps>({
+    route: 'events-facets-stats',
+    eventView,
+    location,
+    orgSlug: organization.slug,
+    getRequestPayload: () => ({
+      ...sortedFacetStatsEventView.getEventsAPIPayload(location),
+      aggregateColumn: 'transaction.duration',
+    }),
+  });
+
+  if (isLoading) {
+    return <Placeholder height="400px" />;
+  }
+
+  const transformedData: DataRow[] = [];
+
+  const totals = data?.totals;
+  const keys = Object.keys(totals);
+  for (let index = 0; index < keys.length; index++) {
+    const element = keys[index];
+    transformedData.push({
+      tagKey: element.split(',')[0],
+      tagValue: element.split(',')[1],
+      throughput: transformSeries('throughput', data![element]['count()'].data),
+      p50: transformSeries('p50', data![element]['p75(transaction.duration)'].data),
+      tpmCorrelation: categorizeCorrelation(totals[element].sum_correlation),
+    });
+  }
+
+  return (
+    <GridEditable
+      isLoading={isLoading}
+      data={transformedData}
+      columnOrder={COLUMN_ORDER}
+      columnSortBy={[]}
+      location={location}
+      grid={{
+        renderBodyCell: (column: GridColumnHeader, row: DataRow) =>
+          renderBodyCell(column, row),
+      }}
+    />
+  );
+}
+
+function categorizeCorrelation(correlation: number): string {
+  if (correlation >= 0.8) {
```

Comment:
I think we may need to wrap these strings in a `t()` call

Reply:
code crimes, we'll just clean it up later!